controllers/jobset_controller.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
we need to move this under pkg, we can do it as follow up.

Reply:
I went ahead and just moved it really quick